BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php

```diff
@@ -72,6 +72,13 @@ public function load(array $configs, ContainerBuilder $container)
         if ('test' == $container->getParameter('kernel.environment')) {
             $loader->load('testservices.yml');
         }
+
+        $availableColor = array_merge($config['available_color'], array(
```

Comment:
I think there's a problem here : The colors should always come from the configuration, not being hardly coded as here (red, green, orange)